explainer.md

```diff
@@ -320,23 +336,183 @@ input.accessibleNode.activeDescendant = optionList;
+console.log(button.accessibleNode.label);  // null
+```
+
+Similarly, setting Accessible Properties on an AccessibleNode does not
+affect corresponding ARIA attributes at all.
+
+```js
+button.accessibleNode.role = "menuitem";
+button.accessibleNode.label = "Next Page (Ctrl+N)";
+console.log(button.getAttribute("role"));  // still "link"
+console.log(button.getAttribute("aria-label"));  // still "Next Page"
+```
+
+Part of the reasoning behind this is that there are some cases in which
+Accessible Properties are more expressive than ARIA attributes - for example,
+referencing elements that don't have IDs. Since there would be no way to
+represent those cases as DOM attribute.
+
+Another reason is for efficiency and tidiness: setting a DOM attribute is more
+heavyweight than setting a property of an object from JavaScript. Authors who
+are concerned about the performance of their web application or the bloated
+size of their generated HTML will find using the Accessibility Object Model
+to be a more efficient alternative. It also promotes creating custom elements
+that are clean and fully encapsulate their own accessibility rather than
+leaking it.
+
+#### Validation
+
+Querying an Accessible Property can be used as a limited
+form of validation and feature detection. In particular, accessing an
+unsupported accessible property returns `undefined`, and accessing a supported
+but unset or invalid accessible property returns `null`.
+
+Let's walk through this in detail.
+
+On an uninitialized AccessibleNode, properties that are supported by the
+browser will return `null` initially, while unsupported / unrecognized
+properties return `undefined`:
+
+```js
+console.log(el.accessibleNode.role); // null
+console.log(el.accessibleNode.label); // null
+console.log(el.accessibleNode.bodyMassIndex); // undefined
+```
+
+When setting the value of an accessible property, that value can
+be subsequently retrieved, but only if set to something valid:
+
+```js
+el.accessibleNode.role = "button";
+console.log(el.accessibleNode.role); // "button"
+
+el.accessibleNode.role = "key";
+console.log(el.accessibleNode.role); // null
+```
+
+Note that "key" is not one of the valid roles in the ARIA spec.
+If a later spec added "key" as a valid role, and a particular
+browser supported it, that string would be returned. This gives
+web developers a way to do some feature detection and learn
+exactly which accessible properties are supported in a particular
+browser version.
+
+Note also that invalid types will also result in returning null.
+For example, rowIndex must be an integer, so trying to set it to a
+string or a floating-point value results in null too.
+
+```js
+cell.accessibleNode.rowIndex = 5;
+console.log(cell.accessibleNode.rowIndex); // 5
+cell.accessibleNode.rowIndex = 5.7;
+console.log(cell.accessibleNode.rowIndex); // null
+cell.accessibleNode.rowIndex = "5";
+console.log(cell.accessibleNode.rowIndex); // null
+```
+
+#### Computed accessible properties
```

Comment:
I'm not super happy with this:
- We're still conflating ARIA reflection with computed property reflection
- I really think the phase 4 discussion is the appropriate place to explain _why_ computed attributes are separate (people can click the link to that phase if they're curious)
- At the beginning of this section, it's not clear why we're suddenly talking about phase 4 when this is a discussion of phase 1.